packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -179,6 +180,29 @@ class WebAssetServer implements AssetReader {
     _hotRestartGeneration++;
   }
 
+  /// Given a list of [modules] that need to be reloaded, writes a file that
+  /// contains a list of objects each with two fields:
```

Comment:
Personally I find it helpful to just see an example of the json output in the comment.

Reply:
Added a json code block here and for `performRestart`.